reference/api/search.md

```diff
@@ -123,16 +122,15 @@ This route should only be used when no API key is required. If an API key is req
 | **[q](/reference/features/search_parameters.md#query-q)**                                     | Query string                                                                                    |    `""`     |
 | **[offset](/reference/features/search_parameters.md#offset)**                                 | Number of documents to skip                                                                     |     `0`       |
 | **[limit](/reference/features/search_parameters.md#limit)**                                   | Maximum number of documents returned                                                            |     `20`      |
-| **[filters](/reference/features/search_parameters.md#filters)**                               | Filter queries by an attribute value                                                            |    `null`     |
-| **[facetFilters](/reference/features/search_parameters.md#facet-filters)** | Facet names and values to filter on                                                                                |    `null`     |
+| **[filter](/reference/features/search_parameters.md#filter)**                               | Filter queries by an attribute value                                                            |    `null`     |
```

Comment:
> Filter queries by an attribute value

Not sure this is accurate. maybe `an attribute sorting condition` ?

Reply:
As I mentioned in another reply, I'm not a fan of `sorting condition` because I'm afraid it only makes sense to a fairly advanced user. As in, a beginner or a user only somewhat familiar with MeiliSearch might look at that and go "WTH is a sorting condition? I want to filter, not sort stuff."

What do you think about:

1. `Filter results using an attribute`

or

2. `Filter results using field attributes and values`